src/content/introduction.md

```diff
@@ -40,8 +40,12 @@
 
 ---
 
+![Redux Flow](content/images/full-redux-flow.png "Redux Flow")
+
+---
+
 ## Not in this talk
-- Redux 101
+- Intro to Redux
```

Comment:
Should we try to briefly explain the concept using one of the diagrams? What do you think @danielfigueiredo ?